.github/actions/deploy-environment/action.yml

```diff
@@ -47,15 +47,47 @@ runs:
       run: |
         access_url=$(terraform -chdir=terraform/application output -json urls | jq -r '.[0]')
         echo "ACCESS_URL=$access_url" >> $GITHUB_OUTPUT
+        check_url=$(terraform -chdir=terraform/application output -json urls | jq -r '.[1]')
+        echo "CHECK_URL=$check_url" >> $GITHUB_OUTPUT
+
+        access_external_domain=$(terraform -chdir=terraform/application output -raw local.access_external_domain 2>/dev/null || echo "")
```

Comment:
this doesn't exist as an output? I would think it best to add the external urls to the urls output in output.tf, preferably removing any duplicates. Or maybe have urls and external_urls as separate outputs?